module/library/categories/helper.js

```
@@ -164,4 +138,192 @@ module.exports = class libraryCategoriesHelper {
+        };
+      }
+      return resolve(createProgramAndSolution)
+    }catch (error) {
+      return resolve({
+        success: false,
+        message: error.message,
+        data: {},
+      })
+
+    }
+  })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)